packages/design-system/src/components/text.tsx

```diff
@@ -47,6 +47,20 @@ export const Text = styled("div", {
         textIndent: "-.025em",
       },
     },
+    weight: {
```

Comment:
would it make sense to have semantic names?

https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping

I would even define those in stitches config and use them everywhere, we need @taylornowotny to define all font weights we are going to need with the corresponding semantic names, maybe he already did that?

Reply:
in the design system ✅

Reply:
we just discussed with taylor, the plan is to use semantic variants for this, so we need to actually rewrite this  entire component to use semantic definitions from the Typography section in DS

we won't need sizes,  or weights

Reply:
Should I change this to semantic names?

Reply:
lets do this in a separate PR, because we need to migrate to an entirely new Text component all usages of it. By semantic I mean variants defined in figma: regular, label, title etc

Reply:
there isn't much point though adding the weight variant to Text right now if we are replacing it entirely

Reply:
oki, gonna resolve this then